unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -18,10 +18,10 @@ use pest::Parser;
 use rand::Rng;
 use semver::Version;
 
-#[cfg(feature = "hostname")]
-use std::env;
 #[cfg(feature = "hostname")]
 use hostname;
+#[cfg(feature = "hostname")]
+use std::env;
```

Comment:
Seems like an arbitrary change? rustfmt update since last time?

Reply:
Yeah it is random, damn rustfmt chose violence this morning